Upstream-Status: Pending

System libdrm doesn't define DRM_FORMAT_INVALID and DRM_FORMAT_010.

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 6e5b65fa806a3b16580df4e0da4333723fd46c3e Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Wed, 16 Sep 2020 16:50:31 +0300
Subject: [PATCH] Revert "ui/gfx/linux: Remove 2 unnecessary preprocessor defs"

This reverts commit 35695844f36a0a849890f785dfdb8b65d17955ba.
---
 ui/gfx/linux/drm_util_linux.cc | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/ui/gfx/linux/drm_util_linux.cc b/ui/gfx/linux/drm_util_linux.cc
index e1372b680926..7482722b1289 100644
--- a/ui/gfx/linux/drm_util_linux.cc
+++ b/ui/gfx/linux/drm_util_linux.cc
@@ -8,6 +8,15 @@
 
 #include "base/notreached.h"
 
+#ifndef DRM_FORMAT_INVALID
+// TODO(mcasas): Remove when uprevving //third_party/libdrm.
+#define DRM_FORMAT_INVALID 0
+#endif
+
+#ifndef DRM_FORMAT_P010
+#define DRM_FORMAT_P010 fourcc_code('P', '0', '1', '0')
+#endif
+
 namespace ui {
 
 int GetFourCCFormatFromBufferFormat(gfx::BufferFormat format) {
-- 
2.25.1

